docs/dev/release_process.rst

```diff
@@ -39,6 +39,31 @@ Create a release branch
 If this is a new major or minor release, you need to make a new branch as described above.
 
 
+Update any translation files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If string interface text has changed, or more complete translations are available, translation files should be updated.
+This is currently done by running the ``make downloadmessages`` command. Following this, the specific files that have been updated with approved translations will need to be added to the repository.
+
+Caveat: by default Crowdin will download all translations, not just approved ones, and will often download untranslated strings also. Do not just add all the files that are downloaded when ``make downloadmessages`` is run, as this will lead to untranslated and poor quality strings being included.
```

Comment:
I've had a lot of trouble in the past with these commands. Maybe we should add a `..warning` that in general "your mileage might vary", and:

* they require java, so you may need to run them in an ubuntu VM
* you might need to manually install the crowdin debian package if the jar isn't working for you
* they might not be compatible with non-bash shells
* you might be better off composing the `crowdin` commands manually

Reply:
also, did we ever figure out why @aronasorman and I weren't able to upload? was it because we didn't run `make makemessages` first?

Reply:
No, you definitely did that, and it still didn't work, but then I did it, and it did :/